src/index.js

```diff
@@ -486,43 +488,43 @@ class EventsCoreAPI {
   }
 
   /**
-   * Authenticating events by verifying signature
+   * Authenticating events by verifying hmac signature
    *
    * @param {object} event JSON payload delivered to the registered webhook URL
    * @param {string} clientSecret Client secret can be retrieved from the Adobe I/O Console integration
-   * @param {string} signatureHeaderValue Value of x-adobe-signature header in each POST request to the registered webhook URL
+   * @param {string} deprecatedSignature Value of x-adobe-signature header in each POST request to the registered webhook URL
    * @returns {boolean} If signature matches return true else return false
```

Comment:
Can we please add the `@deprecated` tag as well?

Reply:
done